### ISSUE TITLE
Quando se cria um paciente deveria aparecer os campos para criar as informações generalistas

### ISSUE THREAD

**Issue by pedroaugustomartins** [pedro-borges/psyNote#47]
Os detalhes não podem ficar tão escondido ao ponto do utilizador não conseguir fazer isso intuitivamente.

**Comment by pedro-borges** [comment]
Que tal navegar automaticamente para a janela de detalhe do paciente?

**Comment by pedroaugustomartins** [comment]
Sim, mas isso só será necessário quando se cria um cliente. Certo?

No dia sexta-feira, 24 de abril de 2015, Pedro Borges <
notifications@github.com> escreveu:
